src/main/java/org/folio/circulation/domain/policy/FixedScheduleRenewalDueDateStrategy.java

```diff
@@ -12,8 +12,7 @@
 
 class FixedScheduleRenewalDueDateStrategy extends DueDateStrategy {
   private static final String NO_APPLICABLE_DUE_DATE_SCHEDULE_MESSAGE =
-    "renewal date falls outside of the date ranges " +
-      "in the fixed schedule of fixed loan policy";
+    "renewal date falls outside of date ranges in fixed loan policy";
```

Comment:
Is there a corresponding pull request waiting for ui-user, as it appear there is behaviour coupled to this message: https://github.com/folio-org/ui-users/blob/master/src/components/Loans/OpenLoans/helpers/isOverridePossible/overridePossibleMessages.js

Reply:
Not for now, I think it will be created later this week (probably tomorrow). Are they expected to be merged at the same time?

Reply:
@dmytrotur Is it necessary to merge them together in order to maintain the current UI behaviour (of showing the correct options depending upon specific message)?

I think changes to the renewal tests in (https://github.com/folio-org/platform-core) likely need to be merged at the same time in order to keep the UI regression tests passing. I'm discussing this with @zburke at the moment

Reply:
@marcjohnson-kint Yes, I think so.

Reply:
@dmytrotur Then I think the UI users and mod-circulation changes should be synchronised to reduce the window where this functionality breaks.

Reply:
@marcjohnson-kint Ok, so I'll let you know, when work in ui-users is completed.

Reply:
@dmytrotur Thank you

Reply:
@marcjohnson-kint The pull request with changes on UI has been created and approved: https://github.com/folio-org/ui-users/pull/734

Reply:
I think we need to also check if the UI regression tests are affected. @zburke we chatted briefly about this yesterday. What are your thoughts?

Reply:
@OleksandrAntonenko1 already has an update ready to go in folio-org/platform-core/pull/227. We'll merge that one immediately after this one.